src/asciidoctorWebViewConverter.ts

```diff
@@ -34,13 +48,27 @@ export class AsciidoctorWebViewConverter {
    * @param transform   An optional string transform that hints at which transformation should be applied to this node
    * @returns           Converted node
    */
-  convert (node, transform) {
+  convert (node, transform): any {
     const nodeName = transform || node.getNodeName()
     if (nodeName === 'inline_anchor' && node.type === 'link') {
       const href = isSchemeBlacklisted(node.target) ? '#' : node.target
       const id = node.hasAttribute('id') ? ` id="${node.id}"` : ''
       const role = node.hasAttribute('role') ? ` class="${node.role}"` : ''
       const title = node.hasAttribute('title') ? ` title="${node.title}"` : ''
+      const sourceInfo = this.getBlockLocation(node)
+      const lineNo = sourceInfo.lineno
```

Comment:
Since `getBlockLocation` can (potentially) return `null`, we should check before calling `sourceInfo.lineno`

Reply:
Quite right.